packages/one-app-bundler/webpack/loaders/provided-externals-loader.js

```diff
@@ -15,25 +15,40 @@
 const loaderUtils = require('loader-utils');
 
 function providedExternalsLoader(content) {
-  const options = loaderUtils.getOptions(this);
-  const providedExternals = options.providedExternals.map((externalName) => {
+  const { moduleName, providedExternals } = loaderUtils.getOptions(this);
+
+  const extendedProvidedExternals = (Array.isArray(providedExternals)
+    ? providedExternals : Object.keys(providedExternals)).map((externalName) => {
     // eslint-disable-next-line global-require, import/no-dynamic-require -- need to require a package.json at runtime
-    const { version } = require(`${externalName}/package.json`);
-    return `'${externalName}': { version: '${version}', module: require('${externalName}')}`;
-  });
+    const externalPkg = require(`${externalName}/package.json`);
+
+    return `
+      '${externalName}': {
+        ...${JSON.stringify({
```

Comment:
using `JSON.stringify` could lead to the same issues as [below](https://github.com/americanexpress/one-app-cli/pull/536/files#diff-6ce8d306a082eb2fdbd92c4b7c26c74b0fcd79c725fa55bcbd2cbd71ee90cd0fR43) with future updates